python-sdk/tests/test_example_dags.py

```diff
@@ -47,47 +47,23 @@ def session():
     return get_session()
 
 
-@pytest.mark.parametrize(
-    "dag_id",
-    [
-        "example_amazon_s3_postgres",
-        "example_amazon_s3_postgres_load_and_save",
-        "example_amazon_s3_snowflake_transform",
-        "example_google_bigquery_gcs_load_and_save",
-        "example_snowflake_partial_table_with_append",
-        "example_sqlite_load_transform",
-        "example_append",
-        "example_load_file",
-        "example_transform",
-        "example_merge_bigquery",
-        "example_transform_file",
-        "calculate_popular_movies",
-    ],
+dag_bag = DagBag(Path.cwd().parent / "example_dags")
```

Comment:
Can we assert that this (`dag_bag.dags`) is not empty, please?

Reply:
https://github.com/astronomer/astro-sdk/actions/runs/3183804425/jobs/5191462176 is failing -- I haven't checked if it is related or not

Reply:
It was related. Fixed in c99e549.

Reply:
The issue was that `cwd` points to the `python-sdk` - not the tests folder. We always run from the project directory e.g. `python-sdk` or `sql-cli`.

And actually `assert dag_bag.dags` wouldn't have helped because it was using the airflow default example dags.

Reply:
WDYT of af23cb3 instead of `.cwd`?

Reply:
Sounds good, yeah. Similar point -> https://github.com/astronomer/astro-sdk/pull/992#discussion_r987802528

How can we protect if one of our example dags isn't a valid DAG, should we validate the # of files equal to # of dags? or is that too brittle, any other thoughts?

Reply:
I would not check for `# of files equal to # of dags` as some files could have more than one dag. one file = one dag is more of a best practice than a rule.

Reply:
I have added a suggestion to the comment you linked.